luigi/rpc.py

```diff
@@ -141,7 +145,7 @@ def _request(self, url, data, log_exceptions=True, attempts=3, allow_null=True):
         body = {'data': json.dumps(data)}
 
         for _ in range(attempts):
-            page = self._fetch(url, body, log_exceptions, attempts)
```

Comment:
What? did the old code retry 3*3 times?? I see both an outer `for _ in range(attempts)` loop and a loop inside the `_fetch` function. Strange...